README.md

```diff
@@ -1,11 +1,11 @@
-<div align="center">
+<!-- <div align="center"> -->
 
 # Diffusers-Interpret 🤗🧨🕵️‍♀️
 
 ![PyPI Latest Package Version](https://img.shields.io/pypi/v/diffusers-interpret?logo=pypi&style=flat&color=orange) ![GitHub License](https://img.shields.io/github/license/JoaoLages/diffusers-interpret?logo=github&style=flat&color=green) 
 
 `diffusers-interpret` is a model explainability tool built on top of [🤗 Diffusers](https://github.com/huggingface/diffusers)
-</div>
+<!-- </div> -->

```

Comment:
I'm not sure why, but the center alignment syntax isn't displayed properly in Mkdocs. I'm temporarily disabling this and will look for a solution later.

Reply:
Yes, we should fix this before merging the PR 👍

Reply:
If we read from `docs/source`, this will not be an issue